src/translator/aggregate_batching_pool.cpp

```diff
@@ -0,0 +1,33 @@
+
+#include "aggregate_batching_pool.h"
+
+namespace marian {
+namespace bergamot {
+
+AggregateBatchingPool::AggregateBatchingPool() {
+  // TODO(@jerinphilip): Set aggregate limits
+}
+
+size_t AggregateBatchingPool::enqueueRequest(Ptr<TranslationModel> model, Ptr<Request> request) {
+  model->enqueueRequest(request);
+  aggregateQueue_.push(model);
```

Comment:
If I understand correctly, a model can be enqueued multiple times.  So if there's a continuous stream of requests coming in, the first element of `aggregateQueue_` will never be popped and `agreegateQueue_` will grow unbounded.

Reply:
This is replaced with an `unordered_set`.